src/lib.rs

```diff
@@ -8,19 +8,24 @@
 pub mod buf;
 pub mod driver;
 pub mod fs;
-mod key;
+
 pub mod net;
 pub mod op;
 
-pub(crate) use key::Key;
+cfg_if::cfg_if! {
```

Comment:
Don't use `cfg_if` if there's only one branch.

Reply:
What's your suggestion

Reply:
See #34 

I also fixed this.